### PR TITLE
Images input

### PR DESCRIPTION
**Pull request by dipen2000**
I created this images and input components please review and give feedback to my PR.

### CODE REVIEW

File: CSS/documentation.css [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+    scroll-behavior: smooth;
+}
+
+
 .container{
     /* border : 3px solid red;  */
-    width : 85%;
-    max-width : 130rem;
+    width : 90%;
+    /* max-width : 130rem; */
```
Comment (dev-ashishsethi) [inline CSS/documentation.css:15]: remove commented code

File: CSS/documentation.css [hunk 2]
```diff
@@ -119,14 +144,37 @@ main{
...
+    /* border : 3px solid red; */
+    display : flex;
+    flex-direction: row;
+    align-items : center;
+    justify-content : center;
+    flex-wrap: wrap;
+    gap : 2rem;
+    padding : 1.5rem;
+    background-color : rgba(102, 51, 153, 0.356);
+    border-top-left-radius: 15px;
```
Comment (dev-ashishsethi) [inline CSS/documentation.css:159]: if size is greater than 10 px then use rem

**Review by dev-ashishsethi — APPROVED** [review]

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    width: 98%;
+}
+
+.textarea-btn{
+    align-self: flex-end;
+}
+
+.form{
+    width : 30rem;
+    /* border : 3px solid red; */
```
Comment (rahulyadav139) [inline components/input/input.css:32]: remove unnecessary comments, use rem.

**Review by rahulyadav139 — APPROVED** [review]

File: components/input/input.html [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                            </div>
+                        </section>
+
+                        <div class="divider mb-2"></div>
+                        <h2 class="mb-2" id="error-form-input">Error form input</h2>
+                        <section class="component-scetion mb-2">
+                            <div class="component-display">
+                                <form class="form">
+                                    <span class="error-msg">Enter correct email and password</span>
+                                    <input
```
Comment (aiman-ansari) [inline components/input/input.html:174]: you can also add "required" it will check whether the field is empty or not.

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
As a suggestion you can make two different container for demo and code snippet it will look good.
Also remove commented code in css file.

**Review by j24m — APPROVED** [review]

File: components/input/input.html (hunk 4 above)
Comment (dipen2000) [reply components/input/input.html:174]: Ok i will add , thanks for your suggestion.

**Pull request merged**